src/Morphir/Visual/DecisionTable.elm

```diff
@@ -42,3 +47,26 @@ which is a function that takes some input and returns a boolean.
 type Match
     = Pattern TypedPattern
     | Guard TypedValue
+
+
+displayTable : DecisionTable -> Element msg
+displayTable table =
+    tableHelp table.decomposeInput table.rules
+
+
+tableHelp : List TypedValue -> List ( List Match, TypedValue ) -> Element msg
+tableHelp headerFunctions rows =
+    Element.none
+
+
+
+{- getDecomposedValues : List TypedValue -> List TypedValue -> List TypedValue
+   getDecomposedValues input decomposeFunction =
+       case input of
+           [] ->
+               []
+           head :: tail ->
+               case decomposeFunction of
+                   decompHead :: decompTail ->
+
+-}
```

Comment:
These changes don't seem to be doing anything or used anywhere. Can you take them out?